packages/create-package/index.js

```diff
@@ -115,7 +115,6 @@ const createWebcomponentsPackage = async () => {
 			"lint": "wc-dev lint",
 			"start": "wc-dev start",
 			"watch": "wc-dev watch",
-			"serve": "wc-dev serve",
```

Comment:
The whole "create package" lib needs to be updated, probably this was commented by mistake?

`main` and `fiori` will not be able to be served individually, but third-party packages should be